src/mscorlib/src/Internal/Runtime/Augments/EnvironmentAugments.cs

```diff
@@ -17,5 +18,11 @@ public static class EnvironmentAugments
         public static bool HasShutdownStarted => Environment.HasShutdownStarted;
         public static string StackTrace => Environment.StackTrace;
         public static int TickCount => Environment.TickCount;
+        public static string GetEnvironmentVariable(string variable) => Environment.GetEnvironmentVariableCore(variable);
```

Comment:
> As we're not intending to use the Win32 version from core

The structure has to be same for all OSes (ie same for Windows and Unix). There is enough of a complexity with where the implementations live already. I do not want to think about and explain to people that which repo the implementation lives in also differs between Windows and Unix.